polyglot.js

```diff
@@ -168,14 +162,28 @@ class PolyGlot {
         select.val(selectedLanguage);
     }
 
-    randomRune() {
-        return String.fromCharCode(Math.floor(Math.random() * 0x51) + 0x16A0)
+    generateRune(string,salt) {
+        var hash = 0;
+        salt = string+salt;
+        for (var i = 0; i < salt.length; i++) {
+            var char = salt.charCodeAt(i);
+            hash = ((hash << 5) - hash) + char;
+            hash = hash & hash;
+        }
+        var randomgen = new MersenneTwister(hash);
+        function randomize() {
+            var char = Math.floor(randomgen.random()*62+48);
+            if (char>57) char+=7;
+            if (char>90) char+=6;
+            return String.fromCharCode(char)
+        }
```

Comment:
So, there's a reason why you generate 62 characters in the specific ranges of 48-57, 65-90 and 97-122 (0-9, A-Z, a-z, in the ascii table). But I'd like a comment to point it out, I'm not a fan of how you did it here.
Note that you could also do it this way : `return Math.floor(randomgen.random()*36).toString(36)` but you lose uppercase, in which case you could do another random to decide if you want to case toUpperCase before returning.
Second note: I don't like declaring that function inside of the function. Use an arrow function instead in the replace